release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -574,7 +574,7 @@ For more information on converting existing ICSP YAML files to IDMS YAML files,
 
 [id="ocp-4-13-toolbox-deprecation"]
 ==== Toolbox is deprecated in {op-system}
-The toolbox library is deprecated and support will be removed from a future {product-title} release.
+The toolbox script is deprecated and support will be removed from a future {product-title} release and replaced with the sos package.
```

Comment:
Well, we're not replacing it with the sos package. We either won't replace it at all or replace it with the containers/toolbox one but it's not decided yet.

```suggestion
The toolbox script is deprecated and support will be removed from a future {product-title} release.
```